lib/active_merchant/billing/gateways/rapyd.rb

```diff
@@ -158,8 +160,14 @@ def add_creditcard(post, payment, options)
         pm_fields[:number] = payment.number
         pm_fields[:expiration_month] = payment.month.to_s
         pm_fields[:expiration_year] = payment.year.to_s
-        pm_fields[:cvv] = payment.verification_value.to_s
         pm_fields[:name] = "#{payment.first_name} #{payment.last_name}"
+
+        if stored_credential
+          pm_fields[:recurrence_type] = stored_credential[:reason_type] if stored_credential[:reason_type]
```

Comment:
I can't find `recurrence_type` in the Rapyd docs. Can you provide a link? [I do see an `initiation_type` field but it lives outside the `payment_method` object](https://docs.rapyd.net/build-with-rapyd/reference/payment-object#payment-method-data-object). Are either of these fields required? I don't really follow the example provided in the Jira description which shows `"name": "recurrence_type",`.

Reply:
I used the field from jira description but I think this might have been mistyped as it should be `initiation_type` as per the documentation, so I have updated it to the correct format. Thanks